desktop_version/src/Game.cpp

```diff
@@ -4812,20 +4812,18 @@ bool Game::savestats(const bool stats_only /*= true*/)
 
     serializesettings(dataNode);
 
-    bool success = FILESYSTEM_saveTiXml2Document("saves/unlock.vvv", doc);
-
-    if (!stats_only)
-    {
-        success &= savesettings();
-    }
+    return FILESYSTEM_saveTiXml2Document("saves/unlock.vvv", doc);
+}
 
-    return success;
+bool Game::savestatsandsettings()
+{
+    return savestats() && savesettings();
```

Comment:
These may need to be or'd so settings aren't skipped when stats fail to save.

Reply:
Ah, good point. Fixed and rebased.